src/nls/messages_en.json

```diff
@@ -66,6 +66,17 @@
     "dashboard.taskRun.status.running": "Running",
     "dashboard.taskRun.status.succeeded": "Completed",
     "dashboard.taskRun.status.waiting": "Waiting",
+    "dashboard.triggerBinding.dateCreated": "Date Created:",
```

Comment:
We've recently removed colons from labels in other parts of the UI, should we do the same here? Might be OK since this is presented differently

Reply:
Only reason I add the colons in this one is because values are appearing to the right of the label.